ui/src/app/tags/views/TagList/TagTable/TagTable.tsx

```diff
@@ -224,7 +222,7 @@ const TagTable = ({
                         )
                       )}
                       <br />
-                      <a href="#todo">
+                      <a href="https://maas.io/docs/how-to-work-with-tags#heading--automatic-tags">
```

Comment:
Missing heading reference

Reply:
> Missing heading reference

Sorry, these pages haven't been updated yet, didn't expect you to jump on this so quickly. 😄  @amylily1011 do you know what's the ETA on this?

Reply:
I got all the screenshots now, will go through the content with Bill tonight. @anthonydillon Can we skip this part for now?